docs/en_US/ExperimentConfig.md

```diff
@@ -399,6 +399,15 @@ machineList:
   
     __gpuIndices__ is used to specify designated GPU devices for NNI, if it is set, only the specified GPU devices are used for NNI trial jobs. Single or multiple GPU indices can be specified, multiple GPU indices are seperated by comma(,), such as `1` or  `0,1,3`.
 
+  * __maxTrialNumPerGpu__
+  
+    __maxTrialNumPerGpu__ is used to specify the max concurrency trial number on a GPU device.
+    
+  * __useActiveGpu__
+  
+    __useActiveGpu__ is used to specify whether to use a GPU if there is another process. By default, NNI will use the GPU only if there is no another active process in the GPU, if __useActiveGpu__ is set to true, NNI will use the GPU regardless of another processes.
```

Comment:
need to state that this option is not applicable for NNI on Windows.

Reply:
Fixed.